my-app/src/main/java/com/JavaWebServer/app/RESTMethods/Head.java

```diff
@@ -0,0 +1,12 @@
+package com.JavaWebServer.app;
```

Comment:
This should be package com.javawebserver.app.restmethods;

We talked about this a while ago, your folders/packages should be lower case in Java.